nnpdf_data/nnpdf_data/commondata/HERA_CC_318GEV/filter.py

```diff
@@ -0,0 +1,166 @@
+from pathlib import Path
+from dataclasses import dataclass
+import typing
+from typing import List
+import numpy as np
+import pandas as pd
+from os import PathLike
+from fortranformat import FortranRecordWriter
+import yaml
+
+@dataclass 
+class commondata:
```

Comment:
If the filters share similar or even identical parts, you can consider to place it in the `filter_utils` folder and store common parts there.